api/gateway-operator/v2alpha1/controlplane_types.go

```diff
@@ -82,14 +84,13 @@ type ControlPlaneSpec struct {
 // +apireference:kgo:include
 // +kubebuilder:validation:XValidation:message="Extension not allowed for ControlPlane",rule="has(self.extensions) ? self.extensions.all(e, (e.group == 'konnect.konghq.com' && e.kind == 'KonnectExtension') || (e.group == 'gateway-operator.konghq.com' && e.kind == 'DataPlaneMetricsExtension')) : true"
 type ControlPlaneOptions struct {
-	// DataPlane designates the target data plane to configure.
+	// IngressClass enables support for the older Ingress resource and indicates
```

Comment:
```suggestion
	// IngressClass enables support for the Ingress resources and indicates
```

I think the word older is not super clear about the intended meaning